tests/check_consistent.py

```diff
@@ -155,6 +134,7 @@ def check_metadata() -> None:
 
 
 if __name__ == "__main__":
+    assert sys.version_info >= (3, 9)
```

Comment:
```suggestion
    assert sys.version_info >= (3, 9), "Python 3.9+ is required to run this test"
```

The other tests which require a version >3.7 have a note in https://github.com/python/typeshed/blob/master/tests/README.md to that effect; could you also add a note there?

Reply:
Or maybe we can just document that we require 3.9+ for all tests? (Even if some tests still work with older Python versions.)

Reply:
That sounds even better actually, it'll mean we don't have to update the README in the tests directory so often :)